booklibrary.html

```diff
@@ -3,9 +3,59 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   
+    <link rel="stylesheet" href="https://bootswatch.com/4/yeti/bootstrap.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.1/css/all.min.css" 
+    integrity="sha512-xA6Hp6oezhjd6LiLZynuukm80f8BoZ3OpcEYaqKoCV3HKQDrYjDE1Gu8ocxgxoXmwmSzM4iqPvCsOkQNiu41GA=="
+     crossorigin="anonymous" />
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@1,300&display=swap" rel="stylesheet">
     <title>Document</title>
+    <link rel="stylesheet" href="style.css">
 </head>
 <body>
+    
+    <div class="container mt-4">
+        <!-- <button id="toggle-btn" onclick="toggle();">Read</button> -->
+        <h1 class="display-4 text-center text-primary">Books Library</h1>
+        <button href="#" id="add-new-book" class="btn btn-outline-primary"><i class="fas fa-plus"></i>     Add New Book</button>
+    <div class="header-container">
+        <h2>Add New Book<button id="hide" class="btn btn-small btn-ghost"><i class="fas fa-times"></i></button></h2>
+        <form action="" id="book-form" class="form mt-5">
+            <div class="form-control">
+                <label for="title">Title</label>
+                <input type="title" id="title">
+                <small class="small-error">Error message</small>
+            </div>
+            <div class="form-control">
+                <label for="author">Author</label>
+                <input type="author" id="author">
+                <small class="small-error">Error message</small>
+            </div>
+            <div class="form-control">
+                <label for="pages">No of Pages</label>
+                <input type="pages" id="pages">
+                <small class="small-error">Error message</small>
+            </div>
```

Comment:
opps, users should not be able to input non-digit value.
![image](https://user-images.githubusercontent.com/31897434/87455650-b3133680-c5fd-11ea-88a2-8285a026d84d.png)